helpers/all.js

```diff
@@ -3,25 +3,19 @@
 const _ = require('lodash');
 
 /**
- * Yield block only if all arguments are valid
+ * Yield block only if all args are valid
  *
  * @example
  * {{#all items theme_settings.optionA theme_settings.optionB}} ... {{/all}}
  */
 const factory = () => {
-    return function() {
-        var args = [], opts, result;
+    return function(...args) {
 
-        // Translate arguments to array safely
-        for (var i = 0; i < arguments.length; i++) {
-            args.push(arguments[i]);
-        }
-
-        // Take the last argument (content) out of testing array
-        opts = args.pop();
+        // Take the last arg out of args array
+        const opts = args.pop();
```

Comment:
The previous comment said that content was the last argument in the array, is that always the case?

Reply:
Yes, what it was popping out of the arguments array was handlebars options object and it still is doing that.

Reply:
Could we update the comment to clarify that?  Possibly something similar to the previous comment.  It can be nice to have that additional context, especially when just doing something like an arbitrary .pop()

Reply:
fixed.

Reply:
👍